### PR TITLE
add vote offices for other countries

### PR DESCRIPTION
**Pull request by mykiwi**

### CODE REVIEW

File: src/AppBundle/Controller/Api/IntlController.php [hunk 1]
```diff
@@ -17,8 +18,17 @@ class IntlController extends Controller
      * @Route("/postal-code/{postalCode}", name="api_postal_code")
      * @Method("GET")
      */
-    public function indexAction($postalCode)
+    public function postalCodeAction($postalCode)
```
Comment (hhamon) [inline src/AppBundle/Controller/Api/IntlController.php]: can you add the string typehint for the argument please.

File: src/AppBundle/Controller/Api/IntlController.php [hunk 2]
```diff
@@ -17,8 +18,17 @@ class IntlController extends Controller
...
     {
         return new JsonResponse(FranceCitiesBundle::getPostalCodeCities($postalCode));
     }
+
+    /**
+     * @Route("/vote-offices/{countryCode}", name="api_vote_offices")
+     * @Method("GET")
+     */
+    public function voteOfficesAction($countryCode)
```
Comment (hhamon) [inline src/AppBundle/Controller/Api/IntlController.php]: Same here, need typehint.

File: front/services/voteOffice/VoteOfficeFormFactory.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import VoteOfficeForm from './VoteOfficeForm';
+
+export default class VoteOfficeFormFormFactory {
```
Comment (rpg600) [inline front/services/voteOffice/VoteOfficeFormFactory.js]: `VoteOfficeFormFactory`  ?

**Comment by hhamon** [comment]
You can squash the commits and rebase onto master.

**Pull request merged**